runtime/codert_vm/cnathelp.cpp

```diff
@@ -3610,11 +3605,10 @@ fast_jitCheckCast(J9VMThread *currentThread, J9Class *castClass, j9object_t obje
 			slowPath = (void*)old_slow_jitCheckCast;
 		}
 	}
-#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
-	else if (J9_IS_J9CLASS_PRIMITIVE_VALUETYPE(castClass)) {
-		slowPath = (void*)old_slow_jitThrowNullPointerException;
-	}
-#endif /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
+	/* In the future Valhalla checkcast needs to throw exception on
+	 * null restricted checkedType if obj is null,
+	 * see issue https://github.com/eclipse-openj9/openj9/issues/19764
```

Comment:
Please correct the reference to `object` and finish each sentence with a period.